### PR TITLE
Sesorov oop

### PR DESCRIPTION
**Pull request by trifonovDmitry**
This version was added new function - /news. Now you can get fresh news from Yandex.ru about COVID-19. Also was added new class COVID, this class is following the principles of OOP.
Added comments

### CODE REVIEW

File: inline_handle.py [hunk 1]
```diff
@@ -19,16 +23,33 @@
...
+CALLBACK_BUTTON_BLOOD_II = "callback_blood_II"
+CALLBACK_BUTTON_BLOOD_III = "callback_blood_III"
+CALLBACK_BUTTON_BLOOD_IV = "callback_blood_IV"
+
+bot: Bot = Bot(
     token=TOKEN,
     base_url=PROXY,  # delete it if connection via VPN
 )
 
+Covid = CovidNews()
```
Comment (vshampor) [inline inline_handle.py:45]: This is a global variable. Should either use `CAPTIAL_CASE` or `python_case`

File: inline_handle.py [hunk 2]
```diff
@@ -19,16 +23,33 @@
...
+
 
 class InlineKeyboardFactory:  # provides all inline keyboards
     @staticmethod
```
Comment (vshampor) [inline inline_handle.py:49]: A factory class is usually organized in another way. Have a single `create_keyboard` static method, which accepts a string, or an `Enum` argument that actually determines the returned keyboard, and then use it as follows:

```
contrast_kb = InlineKeyboardFactory.create_keyboard("contrast")
corona_kb = InlineKeyboardFactory.create_keyboard("coronavirus")
...
```

File: inline_handle.py (hunk 1 above)
Comment (sesorov) [reply inline_handle.py:45]: Fixed in uno-oop branch

File: inline_handle.py (hunk 2 above)
Comment (sesorov) [reply inline_handle.py:49]: Fixed in uno-oop branch

**Pull request merged**